<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectTaskAdminDialog</class>
 <widget class="QDialog" name="ProjectTaskAdminDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="btnCreateProject">
         <property name="text">
          <string>Create project</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnUpdateProject">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Update project</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDeleteProject">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Delete project</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="lstProjects"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="btnCreateTask">
           <property name="text">
            <string>Create task</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnUpdateTask">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Update Task</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeleteTask">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Delete task</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="lstTasks"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjectTaskAdminDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectTaskAdminDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
